solutions/5_betty.js

```diff
@@ -0,0 +1,26 @@
+// Betty Chow
+
+const solution = (arr) => {
+  let newArr = [];
+  for(let j = 0; j < arr.length; j++) {
```

Comment:
Remove this for loop.  You should now have to convert the strings to all lower case. The upper and lowercase string values are compared according to their ASCII values.